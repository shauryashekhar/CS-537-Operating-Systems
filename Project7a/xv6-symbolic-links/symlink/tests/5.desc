Creation of symlink using relative path
