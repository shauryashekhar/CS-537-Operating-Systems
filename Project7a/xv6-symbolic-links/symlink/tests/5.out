SYMLINK: Symlink return = 0
SYMLINK:.              1 1
SYMLINK:..             1 1
SYMLINK:README         2 2
SYMLINK:cat            2 3
SYMLINK:echo           2 4
SYMLINK:forktest       2 5
SYMLINK:grep           2 6
SYMLINK:init           2 7
SYMLINK:kill           2 8
SYMLINK:ln             2 9
SYMLINK:ls             2 10
SYMLINK:test_5         2 11
SYMLINK:mkdir          2 12
SYMLINK:rm             2 13
SYMLINK:sh             2 14
SYMLINK:stressfs       2 15
SYMLINK:usertests      2 16
SYMLINK:wc             2 17
SYMLINK:zombie         2 18
SYMLINK:console        3 19
SYMLINK:test5.txt      2 20
SYMLINK:testSYM5.txt   2 20
